tests/test_dsl.py

```diff
@@ -0,0 +1,377 @@
+import pytest
+
+from snuba_sdk.column import Column
+from snuba_sdk.conditions import Condition, Op
+from snuba_sdk.dsl.dsl import parse_mql
+from snuba_sdk.function import Function
+from snuba_sdk.metrics_query import MetricsQuery
+from snuba_sdk.timeseries import Timeseries, Metric
+
+
+def test_quoted_mri_name() -> None:
+    dsl = "sum(`d:transactions/duration@millisecond`)"
```

Comment:
I'm not going to make you change all these, but I've been trying to use table tests for most of the SDK. I think these would be more concise in that format (https://github.com/getsentry/snuba-sdk/blob/main/tests/test_metrics_query.py#L16)

Reply:
Updated.